nion/swift/model/DisplayItem.py

```diff
@@ -1310,6 +1310,12 @@ def display_layer_factory(lookup_id: typing.Callable[[str], str]) -> DisplayLaye
 
 
 class DisplayItem(Persistence.PersistentObject):
+    DEFAULT_COLORS = ('#1E90FF', "#FF0000", "#00FF00", "#0000FF", "#FFFF00", "#00FFFF", "#FF00FF", "#888888", "#880000",
+                      "#008800", "#000088", "#CCCCCC", "#888800", "#008888", "#880088", "#964B00")
+                      # While one might this as a set, making this a set destroys the order, and would require that
```

Comment:
Can you simplify this to something like "this list should is ordered" or just remove the comment?